content/en/checkout/reference.md

```diff
@@ -19,8 +19,9 @@ Creates a Checkout Session.
 <!-- TODO: Better way to show this! -->
 
 An object with the following properties:
-- `type` (*required*): `Array` of `string`s representing which [payment type(s)](/concepts/payment-types) this Checkout session should accept. For now, only `['single']` is supported.
-- `payment` (*required*): `Object` describing the payment configuration, with the following properties:
+
+- `type` (_required_): `Array` of `string`s representing which [payment type(s)](/concepts/payment-types) this Checkout session should accept. For now, only `['single']` is supported.
```

Comment:
Can you revert these linter changes so we keep it consistent with the other files (the italics)?